src/main/kotlin/no/nav/syfo/client/PdfgenClient.kt

```diff
@@ -30,11 +31,13 @@ fun createPdfPayload(
     validationResult: ValidationResult,
     pasientFnr: String,
     pasientNavn: String,
-    navnSignerendeLege: String
+    navnSignerendeLege: String,
+    antallVedlegg: Int
```

Comment:
Godtar og bruker Pdfgen denne til noe nå?

Reply:
Ja og nei. padm2pdfgen-biten er ikke helt klar. Skal snakke med TH/Malwina på mandag for å få input på hvordan det bør vises i pdf'en. 

Men etter hva jeg har skjønt igjennom testing, så bryr ikke padm2pdfgen seg om felter den ikke er satt opp til å vise. Så det skal vel ikke føre til noen knekk.